### DISCUSSION TITLE
Do we know what happen to the legacy repo?

### DISCUSSION THREAD

**Discussion by XMLDiva** [Azure/azure-mobile-apps#404] [Q&A] [answered]
I used to have https://github.com/Azure/azure-mobile-apps-net-client book marked to reference at. But seems like it's gone? Does anyone know what may have happened to it? I realize it's very old, but we're still using the mobile sync with our custom server side.

**Comment by adrianhall** [comment, marked as answer]
As referenced in the README - it's in the archive branch of this repository.

**Reply by XMLDiva** [reply]
Thanks! I didn't realize there was an archive branch
